MODULE.bazel

```diff
@@ -21,6 +21,7 @@ bazel_dep(name = "rules_go", version = "0.52.0")
 bazel_dep(name = "rules_java", version = "7.12.2")
 bazel_dep(name = "rules_jvm_external", version = "6.7")
 bazel_dep(name = "rules_proto", version = "7.0.2")
+bazel_dep(name = "aspect_bazel_lib", version = "2.20.0")
```

Comment:
As we're a dependency that people will pull in, can we keep this version as low as possible to avoid forcing users to upgrade `aspect_bazel_lib` unless they want to